Source/MARKRangeSlider.m

```diff
@@ -132,15 +138,29 @@ - (void)layoutSubviews
     CGFloat gap = 1.0;
 
     // Set track frame
-    self.trackImageView.frame = CGRectMake(gap, trackY, width - gap, kMARKRangeSliderTrackHeight);
+    CGFloat trackX = gap;
+    CGFloat trackWidth = width - gap*2;
```

Comment:
It certainly does not matter, but I'm a little bit crazy about that :) Could you please add spaces before and after ```*```